### PR TITLE
Feature/11861 fss tooling implement append access control list

### PR DESCRIPTION
**Pull request by pratibh14837**
code changes for Implement append Access Control List

### CODE REVIEW

File: file-share-service-desktop-client/Modules/Admin/JobViewModels/AppendAclJobViewModel.cs [hunk 1]
```diff
@@ -1,24 +1,131 @@
...
         private readonly AppendAclJob job;
+        private readonly Func<IFileShareApiAdminClient> fileShareClientFactory;
+        private string executionResult = string.Empty;
+        string responseMessage = string.Empty;
+        private bool isCommitting;
+        private bool isExecutingComplete;
+        private readonly ILogger<AppendAclJobViewModel> logger;       
 
-        public AppendAclJobViewModel(AppendAclJob job):base (job)
+        public AppendAclJobViewModel(Func<IFileShareApiAdminClient> fileShareClientFactory, AppendAclJob job, ILogger<AppendAclJobViewModel> logger) : base(job,logger)
```
Comment (crdandoukho) [inline file-share-service-desktop-client/Modules/Admin/JobViewModels/AppendAclJobViewModel.cs:27]: Please position AppendAclJob first in the list of arguments for consistency with other JobViewModels

File: file-share-service-desktop-client/Modules/Admin/JobViewModels/AppendAclJobViewModel.cs [hunk 2]
```diff
@@ -1,24 +1,131 @@
...
-        protected internal override Task OnExecuteCommand()
+
+        public DelegateCommand CloseExecutionCommand { get; }
+
+        private void OnCloseExecutionCommand()
+        {
+            ExecutionResult = string.Empty;
+            IsExecutingComplete = false;
+        }
+        protected internal override async Task OnExecuteCommand()
```
Comment (crdandoukho) [inline file-share-service-desktop-client/Modules/Admin/JobViewModels/AppendAclJobViewModel.cs:46]: Please tidy the indentation in the OnExecuteCommand() method

**Review by crdandoukho — CHANGES_REQUESTED** [review]
Hi Pratibha, just a few minor changes - please see my comments. Please also fix any new compiler warnings that have been introduced by your new code.

File: file-share-service-desktop-client/Modules/Admin/JobViewModels/AppendAclJobViewModel.cs (hunk 1 above)
Comment (pratibh14837) [reply file-share-service-desktop-client/Modules/Admin/JobViewModels/AppendAclJobViewModel.cs]: Resolved

File: file-share-service-desktop-client/Modules/Admin/JobViewModels/AppendAclJobViewModel.cs (hunk 2 above)
Comment (pratibh14837) [reply file-share-service-desktop-client/Modules/Admin/JobViewModels/AppendAclJobViewModel.cs:45]: Resolved

**Review by crdandoukho — APPROVED** [review]
Thanks Pratibha for resolving my review comments.

Looks good. Approved

**Review by martyn-fewtrell — APPROVED** [review]
Looks good

**Pull request merged**